src/Notepads/Strings/en-US/Resources.resw

```diff
@@ -573,4 +573,12 @@
     <value>Encoding cannot be determined</value>
     <comment>TextEditor: Notification message when file's encoding cannot be determined.</comment>
   </data>
+  <data name="FindAndReplace_SearchBackwardButton.ToolTipService.ToolTip" xml:space="preserve">
```

Comment:
Can you add these two newly added entries to all locales? Just use English for all.